org.adempiere.base/src/org/compiere/process/ProcessInfo.java

```diff
@@ -938,5 +942,13 @@ public void setProcessUI(IProcessUI processUI) {
 	public IProcessUI getProcessUI() {
 		return processUI;
 	}
+
+	public boolean isReportOverride() {
+		return m_IsReportOverride;
+	}
+
+	public void setIsReportOverride() {
+		this.m_IsReportOverride = !(MSysConfig.getBooleanValue(MSysConfig.ZK_REPORT_TABLE_OPEN_IN_NEW_TAB, false, Env.getAD_Client_ID(Env.getCtx())));
+	}

```

Comment:
please always add javadoc for public methods. also, as per my comments above, I think it is more readable to use isReplaceTabContent and setReplaceTabContent instead.